packages/sui-test/bin/karma/index.js

```diff
@@ -38,6 +45,11 @@ module.exports = async ({watch, ci, pattern, ignorePattern, srcPattern}) => {
     [pattern]: ['webpack']
   }
 
+  configure.client.mocha = {
+    ...configure.client.mocha,
+    ...(timeout && {timeout})
```

Comment:
👀 it would be enough passing just`timeout`

Reply:
... or set a default value

Reply:
I am not sure what could happen if I pass timeout as undefined to the config.